internal/components/meshexpansion/testdata/icp-test-cr.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: servicemesh.cisco.com/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: cp-v111x
+  namespace: istio-system
+spec:
+  version: "1.11.0"
+  meshExpansion:
+    enabled: true
+    istiod:
+      expose: true
+    webhook:
+      expose: true
+    clusterServices:
+      expose: true
+    gateway:
+      metadata:
+        labels:
+          mgw-label: labelvalue
```

shouldn't this be `label` simply?
```suggestion
          label: labelvalue
```

---

it is just a bogus label name